webhooks/pod_webhook.go

```diff
@@ -74,12 +76,20 @@ func (m *PodMutator) Handle(ctx context.Context, req admission.Request) admissio
 		return admission.Denied("static or orphaned pods cannot be mutated")
 	}
 
-	if code, err := m.handleRPCEvaluation(ctx, req, annotations, pod); err != nil {
-		if code == 0 {
-			return admission.Denied(err.Error())
-		} else {
+	if shouldUseSidecar(annotations) {
+		if code, err := m.handleRPCConfiguration(ctx, req, annotations, pod); err != nil {
+			if code == http.StatusForbidden {
+				return admission.Denied(err.Error())
+			} else {
+				return admission.Errored(code, err)
+			}
+		}
+	} else if shouldUseInProcess(annotations) { // use in-process evaluation
+		if code, err := m.handleInProcessConfiguration(ctx, req, annotations, pod); err != nil {
 			return admission.Errored(code, err)
 		}
+	} else {
+		return admission.Denied("cannot mutate pods without 'featureflagsource' or 'inprocessconfiguration' annotation")
```

Comment:
nit - need to emphasise this is when `openfeature.dev/enabled` annotation is present.
```suggestion
		return admission.Denied("cannot mutate pods without a 'featureflagsource' or 'inprocessconfiguration' annotation as openfeature.dev/enabled annotation is set to true)
```